### PR TITLE
Clarify contributors roles

### PR DESCRIPTION
**Pull request by rdswift**
### Summary

This is a…

- [ ] Correction
- [x] Addition
- [x] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

Needed to clarify the order of the contributors in the list and what was meant by a contributor.  Also needed to elevate the acknowledgements of the contributions.

### Description of the Change

Added explanation of contributors list and order in alphabetical order.  Elevate the acknowledgements section to be even with contributing section so that it appears in the TOC.

### Additional Action Required

None.

### CODE REVIEW

File: source/acknowledgements.rst [hunk 1]
```diff
@@ -39,9 +39,15 @@ Spanish (Lead: TBA)
 * TBA
 
 
-Content Contributors
+Contributors
 --------------------
 
+This list includes contributors to the documentation, regardless of their role.  In addition to actual content contributors,
+this includes leads, translators, reviewers and proofreaders.  All contributions are valuable and appreciated.  As such, the
+names are presented in alphabetical order so as not to imply the relative importance of someone's contribution based on their
```
Comment (phw) [inline source/acknowledgements.rst:47]: Now the names list just needs to be actually sorted alphabetically ;)

Comment (rdswift) [reply source/acknowledgements.rst:47]: They are in alphabetical order based on surname (where available).  Which ones are out of place?  Should I order the list based on given names instead?

Comment (phw) [reply source/acknowledgements.rst:47]: Nah, all good, nevermind. I checked for the surname, but not sure what I saw.

Comment (rdswift) [reply source/acknowledgements.rst:47]: I know that feeling.  After your comment, I went over it three times because I wasn't sure it was right.  :wink:

**Pull request merged**